overrides/assets/stylesheets/home.css

```diff
@@ -352,7 +352,7 @@ h2.secondary-headline, h3.trusted-by {
   justify-content: center;
   color: black;
   display: flex;
-  background-color: white;
+  background-color: #f3f1f2;

```

Comment:
Can you reference one of the css custom properties here instead of a hard coded css value?

Reply:
yeah sure !